tests/CSharpTemp/CSharpTemp.h

```diff
@@ -509,3 +509,27 @@ class DLL_API TestOverrideFromSecondaryBase : public Foo, public SecondaryBase
     void VirtualMember();
     void setProperty(int value);
 };
+
+class DLL_API TestParamToInterfacePassBaseOne
+{
+};
+
+class DLL_API TestParamToInterfacePassBaseTwo
+{
+	int m;
```

Comment:
I think you need to initialize m to 0 since you're checking its value later with an assert.